tests/Doctrine/MongoDB/Tests/Query/BuilderTest.php

```diff
@@ -177,7 +177,6 @@ public function testFindQuery()
     public function testUpsertUpdateQuery()
     {
         $qb = $this->getTestQueryBuilder()
-            ->update()
```

Comment:
You're correct. I've had half a mind of making upserts imply an update but then I changed my mind and forgot to change this back. Fixed now.